### ISSUE TITLE
Error: getaddrinfo ENOTFOUND cdn4.ps1.photosynth.net...

### ISSUE THREAD

**Issue by tophtucker** [dddExperiments/PhotosynthDownloader#9]
Mac OS X El Capitan w/ node v6.9.5, getting an ENOTFOUND error. This feels like maybe a problem on the Photosynth end? Maybe they've already started taking servers down and I'm too late... sigh. Sorry!! : / (And thank you so much for your great and timely utility!)

```
Downloading photosynth: 351243c3-46dc-454f-a53d-6dff34acd2f4 to output
0 comments downloaded.
Downloading 180 images.
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: getaddrinfo ENOTFOUND cdn4.ps1.photosynth.net cdn4.ps1.photosynth.net:80
    at errnoException (dns.js:28:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:76:26)
```

**Comment by tophtucker** [comment]
(N.B. I appear to have it working perfectly on my old Windows machine, so no rush on my account!! Seriously, thank you so so much. You don't know how much it means to me.)

**Comment by dddExperiments** [comment]
I tried on Ubuntu and Windows and didn't get any issue with your synth.
The only thing that sometimes happens is a timeout issue, this is why I wrote my scripts so that they resume exactly where they stopped. Maybe on Mac OS X the timeout issues are manifesting with ENOTFOUND dns error? -> try re-download 2 min later?

I'm glad you managed to save your synths in time and that my scripts are found useful!
